docs/reference/mapping/runtime.asciidoc

```diff
@@ -42,6 +42,37 @@ front, and can use runtime fields to amend the mapping at any time. Using
 runtime fields allows for a smaller index and faster ingest time, which
 combined use less resources and reduce your operating costs.
 
+[discrete]
+[[runtime-performance]]
+=== Performance
```

Comment:
Sorry for being picky :) I wonder if we should focus on why using runtime fields, and change the title of the section. Then we can also address the performance bit as part of it, in comparison to the previous ways of using scripting in the search API

Reply:
Being picky is good in technical documentation 😄  Maybe we call this section **Usage** and move the last section starting with "How you use a runtime field determines the number of documents that the included script runs against." to the beginning. Then, we keep the remaining sections as they are and end with performance comparisons. Thoughts?

Reply:
that sounds good, we could also call it Motivation or anything along these lines. I like the general direction

Reply:
I'm going to go with **Incentives** as the title. Thanks for helping to iterate and hone this section 👍